test/ibmq/test_experiment.py

```diff
@@ -199,6 +199,8 @@ def test_retrieve_experiment(self):
         exp = self.experiments[0]
         rexp = self.provider.experiment.retrieve_experiment(exp.uuid)
         self.assertEqual(exp.uuid, rexp.uuid)
+        for attr in ['hub', 'group', 'project']:
+            self.assertIsNotNone(getattr(rexp, attr), "{} does not have a {}}".format(rexp, attr))
```

Comment:
`test/ibmq/test_experiment.py:203:54: W1302: Invalid format string (bad-format-string)`
```suggestion
            self.assertIsNotNone(getattr(rexp, attr), "{} does not have a {}".format(rexp, attr))
```